Gemfile.lock

```diff
@@ -257,6 +257,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-21
+  ruby
```

Comment:
any reason why this was added?

Reply:
Something that I installed or configured added that automatically. Not sure what-- it may have been something that I tried and backed away from. I will remove it, assuming that doesn't break anything.

Reply:
yeah, don't want to edit the lockfile manually, since as you said, it's a generated file. Any idea what caused an update to the lockfile?

Reply:
No. I can try to go back and reproduce it.